ui/src/reducers/index.js

```diff
@@ -11,12 +18,31 @@ const columns = [
 
 const apps = { rows: [], data: {} };
 
+const appDetailColumns = [
+  { title: 'App Version', transforms: [ sortable ] },
+  { title: 'Current Installs', transforms: [ sortable ] },
+  { title: 'Launches', transforms: [ sortable ] },
+  { title: 'Last Launched', transforms: [ sortable ] },
+  { title: 'Disable on Startup', transforms: [ sortable ] },
+  { title: 'Custom Disable Message', transforms: [ sortable ] }
+];
+
+const appDetailRows = [
```

Comment:
IS this just for testing purposes? I assume we can remove it later?

Reply:
Seed data for the moment, until we have something coming in from an api